include/deal.II/grid/tria.h

```diff
@@ -39,7 +39,7 @@ DEAL_II_ENABLE_EXTRA_DIAGNOSTICS
 #include <vector>
 #include <list>
 #include <map>
-#include <algorithm>
+#include <numeric>
```

Comment:
I don't think you need this anymore. Just rebase to master and this should go away.